### DISCUSSION TITLE
can directus be running at an URL location other than root behind a reverse proxy?

### DISCUSSION THREAD

**Discussion by xu4wang** [directus/directus#11558] [Archive]
I understand the PUBLIC_URL in env file can be changed to PUBLIC_URL="http://directus.mydomain.com"

Is it possible to use PUBLIC_URL="http://app.mydomain.com/directus"  ?

The use case is that directus will be running behind a nginx.   We are planning to proxy pass to the port directus is listening.

```
location ^~ /directus {
       ...
      proxy_pass http://127.0.0.1:5050;

}
```

**Comment by azrikahar** [comment]
This answer should be helpful as it's achieving the same thing with Nginx: https://github.com/directus/directus/discussions/4480#discussioncomment-473036

**Reply by xu4wang** [reply]
cool. it is a good workaround.
